aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -8,41 +8,33 @@ By [Peter Kellner](http://peterkellner.net)
 
 The ```AnchorTagHelper``` enhances the html anchor (`<a ... ></a>`) tag. A new set of attributes are defined that work with the anchor tag. The link generated (on the href tag) is based on a combination of these new attributes that work together to form the final redirect URL. That URL can include an optional protocol such as https.
```

Comment:
points 1 and 2 updated.
I've re-read the passage mentioned a few times and it seems OK to me.  There is another section already written by Rick that introduced TH's.  I don't see anything there that specifically talks about non-standard attributes being converted to standard attributes but I'm not sure that needs to be mentioned (can't hurt to add it there).

My assumption in writing all these built in TH docs it that the reader knows the basics of tag helpers. Valid assumption?

https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/intro